marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-mapping-writer.json

```diff
@@ -1,4 +1,5 @@
 {
   "role-name": "data-hub-mapping-writer",
-  "description": "Permits updating mapping documents"
+  "description": "Permits updating mapping documents",
+  "privilege": []
```

Comment:
I am wondering if this role should gain the data-hub:writeMapping privilege. Then we don't even need to modify data-hub-developer. It would seem surprising if this role doesn't inherit that privilege - thoughts?

Reply:
The only issue I have with that is that if we added the privileges to the lower-level data-hub-* roles then that means that for example when the hub-central-step-runner gets the reader roles necessary the privileges will come over with them. With the current roles defined, we could potentially have the writer privileges on the lower level roles and not the read. 

I'm thinking doing any assert on reading documents would be redundant anyway given the document permissions we're setting on the documents.